### PR TITLE
Enable `kast` tests

### LINKED ISSUE #4: Failing tests in test_kprint
~Blocked on: runtimeverification/k#2796~

We should re-enable these tests: https://github.com/runtimeverification/k/blob/86f098031fccf61e815a195d95d8037088a95495/pyk/src/pyk/integration_tests/test_kprint.py#L24
And this one: https://github.com/runtimeverification/k/blob/86f098031fccf61e815a195d95d8037088a95495/pyk/src/pyk/integration_tests/test_kore_to_kast.py#L23

### PR DESCRIPTION
**Pull request by tothtamas28**
Fixes #4

### CODE REVIEW

File: src/integration_tests/test_kprint.py [hunk 1]
```diff
@@ -23,8 +24,8 @@ def test_parse_token(self) -> None:
             ('int-token', KToken('3', 'Int'), intToken(3)),
             ('id-token', KToken('abc', 'Id'), KToken('abc', 'Id')),
             ('add-aexp', KToken('3 + 4', 'AExp'), KApply('_+_', [intToken(3), intToken(4)])),
-            # TODO: ('add-int', KToken('3 +Int V', 'Int'), KApply('_+Int_', [intToken(3), KVariable('V')])),
-            # TODO: ('k-cell', KToken('<k> . </k>', 'KCell'), KApply('<k>', [KApply(Labels.EMPTY_K)])),
+            ('add-int', KToken('3 +Int V', 'Int'), KApply('_+Int_', [intToken(3), KVariable('V')])),
+            ('k-cell', KToken('<k> . </k>', 'KCell'), KApply('<k>', [KApply(EMPTY_K)])),
```
[the remark this answers is not in the document]
Comment (radumereuta) [reply src/integration_tests/test_kprint.py]: `kast --input rule`
Those cells and +Int are not visible in the program parser.

[the remark this answers is not in the document]
Comment (tothtamas28) [reply src/integration_tests/test_kprint.py]: Thanks, that solved the problem.

Now I see the following failure.

```
% kast --definition definitions/imp --input rule --output json --sort Id --expression 'abc' --debug
org.kframework.utils.errorsystem.KEMException: [Error] Inner Parser: Parse error: unexpected end of file following token 'abc'.
        Source(<command line: -e>)
        Location(1,4,1,4)
        at org.kframework.utils.errorsystem.KEMException.create(KEMException.java:143)
        at org.kframework.utils.errorsystem.KEMException.innerParserError(KEMException.java:96)
        at org.kframework.parser.inner.kernel.EarleyParser.parseError(EarleyParser.java:886)
        at org.kframework.parser.inner.kernel.EarleyParser.parse(EarleyParser.java:847)
        at org.kframework.parser.inner.ParseInModule.parseStringTerm(ParseInModule.java:228)
        at org.kframework.parser.inner.ParseInModule.parseString(ParseInModule.java:193)
        at org.kframework.parser.inner.ParseInModule.parseString(ParseInModule.java:157)
        at org.kframework.kast.KastFrontEnd.run(KastFrontEnd.java:135)
        at org.kframework.main.FrontEnd.main(FrontEnd.java:57)
        at org.kframework.main.Main.runApplication(Main.java:118)
        at org.kframework.main.Main.runApplication(Main.java:108)
        at org.kframework.main.Main.main(Main.java:56)
```

Parsing it as `program` works:

```
% kast --definition definitions/imp --input program --output json --sort Id --expression 'abc' --debug
{"format":"KAST","version":2,"term":{"node":"KToken","sort":{"node":"KSort","name":"Id"},"token":"abc","att":{"node":"KAtt","att":{"org.kframework.attributes.Location":[1,1,1,4],"org.kframework.attributes.Source":"<command line: -e>"}}}}
```

Is this expected?

[the remark this answers is not in the document]
Comment (radumereuta) [reply src/integration_tests/test_kprint.py]: Yes. Ids have very similar regexes with variables, so by default, they are not allowed in the rule parser.

**Review by radumereuta — APPROVED** [review]
One question.
Why is the version bump the first commit?

**Comment by tothtamas28** [comment]
> One question. Why is the version bump the first commit?

I did a rebase.

**Pull request merged**